mars-cli/tests/test_biosample_external_references.py

```diff
@@ -67,7 +67,7 @@ def test_validate_bs_accession():
         validate_bs_accession(invalid_accession)
 
     valid_accession = "SAMEA112654119"
-    assert validate_bs_accession(valid_accession) != ValueError
+    assert validate_bs_accession(valid_accession) is not ValueError
```

Comment:
I would normally just do this to assert no exception is raised, since it will never "return" `ValueError` anyway:
```suggestion
    validate_bs_accession(valid_accession)
```